test/unit/spec/function_output/kbaseAttributeMapping-spec.js

```diff
@@ -58,7 +61,7 @@ define(['jquery', 'kbaseAttributeMapping', 'base/js/namespace', 'narrativeMocks'
                 ],
             };
             // this code is more of less ignored, because two WS calls are made both can't be stubbed
```

Comment:
I remember this bit of code -- if you run it without an internet connection you get workspace connection failures. 😖